wandb/sdk/launch/docker.py

```diff
@@ -240,9 +239,16 @@ def build_docker_image_if_needed(
     )
     dockerfile_contents += env_vars + "\n"
 
+    if runner_type == "gcp-vertex":
+        # supply the entrypoint via the dockerfile since we don't `docker run`
+        dockerfile_contents += 'ENTRYPOINT ["python", "{entrypoint}"]\n'.format(
+            entrypoint=launch_project.get_single_entry_point().name
```

Comment:
Potentially forgetting something here, but won't this discard any additional arguments the user has supplied?